userspace/libscap/engine/savefile/scap_reader.c

```diff
@@ -0,0 +1,92 @@
+/*
```

Comment:
Can we call this file `scap_reader_buffered`? 
Or do we expect to add more scap_readers implementations here sooner or later?

Reply:
Not sure, but in that case it would make sense to have each implementation in its own file.